web/src/views/Login.vue

```diff
@@ -64,6 +82,8 @@ onMounted(async () => {
     return;
   }
 
+  forges.value = (await apiClient.getForges()) ?? [];
```

Comment:
Wouldn't do so. As you noted pinia doesn't store state when doing actual http redirects. Coming back to the login page should be pretty rare in general and adding all those things to a store will just increase the memory consumption. Also we would need to invalidate this cache in case an admin changes forge settings, adds a new one etc.